lib/Extension/Core/ColumnType/DateTime.php

```diff
@@ -153,6 +154,18 @@ private function getInputData($value)
 
                             $inputData[$field] = $value[$field];
                             break;
+                        case 'datetime_interface':
+                            if (interface_exists('\DateTimeInterface')) {
+                                if (!empty($value[$field]) && !($value[$field] instanceof \DateTimeInterface)) {
+                                    throw new DataGridColumnException(
+                                        sprintf('Value in field "%s" is "%s" type instead of "\DateTimeInterface" instance.',
+                                            $field, gettype($value[$field]))
+                                    );
+                                }
+
+                                $inputData[$field] = $value[$field];
+                                break;
+                            }
```

Comment:
Should there not be an exception if you try to use this column type without the interface existing?